examples/direct_fidelity_estimation.py

```diff
@@ -53,22 +50,31 @@ def compute_characteristic_function(circuit: cirq.Circuit,
 
 
 def direct_fidelity_estimation(circuit: cirq.Circuit, qubits: List[cirq.Qid],
-                               noise: cirq.NoiseModel, n_trials: int):
+                               noise: cirq.NoiseModel, sampled: bool,
+                               n_trials: int, samples_per_term: int):
```

Comment:
What should I pass for `samples_per_term` if `sampled=False`?

Reply:
Good point. Removed the flag --sampled and now use only --samples_per_term to decide whether to sample (if > 0).